packages/shared/src/components/squads/SquadsDirectoryHeader.tsx

```diff
@@ -1,52 +1,59 @@
 import React, { ReactElement } from 'react';
+import classNames from 'classnames';
 import { cloudinary } from '../../lib/image';
 import { Button, ButtonVariant } from '../buttons/Button';
 import SourceBetaIcon from '../../../icons/source_beta.svg';
 import { Origin } from '../../lib/analytics';
 import { useSquadNavigation } from '../../hooks';
 
-export const SquadsDirectoryHeader = (): ReactElement => {
+interface SquadsDirectoryHeaderProps {
+  className?: string;
+}
+
+export const SquadsDirectoryHeader = ({
+  className,
+}: SquadsDirectoryHeaderProps): ReactElement => {
   const { openNewSquad } = useSquadNavigation();
 
   return (
-    <div className="mb-4">
```

Comment:
Diff looks crazy, but the major difference is just the removal of this unnecessary wrapper and allowing to pass `className`.